src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -469,6 +468,30 @@ def rm_snapshot(self, fs_id, path, name):
         cfs = self._cephfs_instance(fs_id)
         cfs.rm_snapshot(path, name)
 
+    @RESTController.Resource('GET')
+    def get_quotas(self, fs_id, path):
+        """
+        Get the quotas of the specified path.
+        :param fs_id: The filesystem identifier.
+        :param path: The path of the directory/file.
+        :return: Returns a dictionary containing 'max_bytes' and 'max_files'.
+        :rtype: dict
+        """
+        cfs = self._cephfs_instance(fs_id)
+        return cfs.get_quotas(path)
+
+    @RESTController.Resource('POST')
+    def set_quotas(self, fs_id, path, max_bytes=None, max_files=None):
+        """
+        Set the quotas of the specified path.
+        :param fs_id: The filesystem identifier.
+        :param path: The path of the directory/file.
+        :param max_bytes: The byte limit.
+        :param max_files: The file limit.
+        """
+        cfs = self._cephfs_instance(fs_id)
+        return cfs.set_quotas(path, max_bytes, max_files)
+
```

Comment:
Yup, I agree with @Devp00l. Looks like something is rotten here. This PR shouldn't be adding this logic. If this happens it means it's duplicated somewhere.

Reply:
Yeah, this is all coming from rebasing after the PR that changed some of the endpoint methods; I'm fine with removing it